src/frame/staking.rs

```diff
@@ -111,6 +114,7 @@ impl Default for ValidatorPrefs {
 /// The subset of the `frame::Trait` that a client must implement.
 #[module]
 pub trait Staking: System {}
+impl<T: System> Staking for T {}
```

Comment:
Why this? A runtime that implements system may not impl staking

Reply:
I did this because `Staking` has no new items beyond those in `System`, but you are correct that this impl is too broad.